docs/release-notes.md

```diff
@@ -1,6 +1,37 @@
 # Release notes
 
-## Trino Gateway 10 (24 July 2024)
+## Trino Gateway 11 (4 Sep 2024)
+
+[JAR file gateway-ha-10-jar-with-dependencies.jar](https://repo1.maven.org/maven2/io/trino/gateway/gateway-ha/11/gateway-ha-11-jar-with-dependencies.jar),
+Docker container `trinodb/trino-gateway:11`
+
+* Add support to optionally deactivate adding `X-Forwarded-*` HTTP headers to
+  each request with the `addXForwardedHeaders` configuration in `routing`.
+  ([#417](https://github.com/trinodb/trino-gateway/pull/417))
+* Add OpenMetrics endpoint to
+  [enable monitoring](operation.md#monitoring) with Prometheus and compatible
+  systems.
+  ([#429](https://github.com/trinodb/trino-gateway/pull/429))
+* Add support to [deactivate hostname verification for the certificate of the
+  Trino clusters](security.md#cert-trino).
+  ([#436](https://github.com/trinodb/trino-gateway/pull/436))
+* Add support for specifying custom labels in `commonLabels` with the Helm
+  chart. ([#448](https://github.com/trinodb/trino-gateway/pull/448))
+* Allow using Spanner as backend database.
+  ([#375](https://github.com/trinodb/trino-gateway/pull/375))
+* [:warning: Breaking change:](#breaking) Require Java 22 for build and runtime.
```

Comment:
Should we move the breaking change to the first line?

Reply:
Yeah .. let me do that